src/vic3_world/countries/vic3_country_importer.cpp

```diff
@@ -42,6 +42,16 @@ vic3::BudgetLevel parseBudgetLevel(const std::string& level_string)
 
 vic3::CountryImporter::CountryImporter()
 {
+   dynamic_name_parser_.registerKeyword("dynamic_country_name", [this](std::istream& input_stream) {
+      options_.dynamic_name = commonItems::remQuotes(commonItems::getString(input_stream));
```

Comment:
getString will already do remQuotes. Unless this is some special case of double quotes, which according to tests is doubtful, you don't need it.